pkg/cluster/k8sres.go

```diff
@@ -362,6 +363,26 @@ func (c *Cluster) generatePodTemplate(resourceRequirements *v1.ResourceRequireme
 	if cloneDescription.ClusterName != "" {
 		envVars = append(envVars, c.generateCloneEnvironment(cloneDescription)...)
 	}
+
+	var names []string
+	// handle environment variables from the PodEnvironmentConfigMap. We don't use envSource here as it is impossible
```

Comment:
What is  `envSource` ? I cannot find code that references it.

Reply:
https://kubernetes.io/docs/api-reference/v1.8/#configmapenvsource-v1-core